### PR TITLE
Update Scotland.gov.uk.xml and Create gov.scot.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

**Comment by J0WI** [comment]
The list of `Nonfunctional hosts in *scotland.gov.uk` seems to be outdated too.
Looks like they moved to `gov.scot` anyway?

**Comment by cschanaj** [comment]
@J0WI I have rewritten the ruleset for `gov.scot`, please take a look when you have the time. thanks. :smile:

**Comment by J0WI** [comment]
Well, ideally we keep both :)

**Comment by cschanaj** [comment]
@J0WI fixed! thanks!!

File: src/chrome/content/rules/Scotland.gov.uk.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	For other UK government coverage, see GOV.UK.xml.
+
+	Non-functional hosts
+		Timeout was reached:
+		- scotland.gov.uk
+		- labs.data.scotland.gov.uk
+		- simd.scotland.gov.uk
+		- www.scotland.gov.uk
+
+		SSL peer certificate was not OK:
```
Comment (J0WI) [inline src/chrome/content/rules/Scotland.gov.uk.xml]: Just one more thing: `aquaculture.scotland.gov.uk` got lost.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**